src/js/dataMapping/contracts/additionalDetails.js

```diff
@@ -3,416 +3,230 @@
  * Created by Kevin Li 3/3/17
  */
 
-import * as MoneyFormatter from 'helpers/moneyFormatter';
-
 export const agencyFields = [
     {
         label: 'Awarding Agency',
-        field: 'awarding_agency_name'
+        field: '__special',
+        parse: (data) => data.awardingAgency.name
```

Comment:
When a `parse` function is no longer needed because there is no parsing logic, we should remove the function and update the `field` value to point to the model's property.

Reply:
I'm using parse functions for nested fields when the data are on different levels or in different objects (in this case `awardingAgency` and `fundingAgency`). Let me know if there is a better way to access/ organize them.